week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -92,5 +92,120 @@ function testAll() {
     )
   );
 }
+/**
+Let's peer into the future using a Magic 8 Ball!
+https://en.wikipedia.org/wiki/Magic_8-Ball 
+There are a few steps to being able view the future though:
+* Ask a question
+* Shake the ball
+* Get an answer
+* Decide if it's positive or negative
+The question can be anything, but the answers are fixed,
+and have different levels of positivity or negativity.
+Below are the possible answers:
+## Very positive
+It is certain.
+It is decidedly so.
+Without a doubt.
+Yes - definitely.
+You may rely on it.
+## Positive
+As I see it, yes.
+Most likely.
+Outlook good.
+Yes.
+Signs point to yes.
+## Negative
+Reply hazy, try again.
+Ask again later.
+Better not tell you now.
+Cannot predict now.
+Concentrate and ask again.
+## Very negative
+Don't count on it.
+My reply is no.
+My sources say no.
+Outlook not so good.
+Very doubtful.
+*/
+
+// This should log "The ball has shaken!"
+// and return the answer.
+const answers = ["It is certain", "It is decidedly so", "Without a doubt", "Yes - definitely", "You may rely on it",
+"As I see it, yes","Most likely", "Outlook good", "Yes", "Signs point to yes",
+"Reply hazy, try again", "Ask again later", "Better not tell you now", "Cannot predict now", "Concentrate and ask again",
+"Don't count on it", "My reply is no", "My sources say no", "Outlook not so good", "Very doubtful"];
+
+const veryPositive = ["It is certain", "It is decidedly so", "Without a doubt", "Yes - definitely", "You may rely on it"];
+const positive = ["As I see it, yes","Most likely", "Outlook good", "Yes", "Signs point to yes"];
+const negative = ["Reply hazy, try again", "Ask again later", "Better not tell you now", "Cannot predict now", "Concentrate and ask again"];
+const veryNegative = ["Don't count on it", "My reply is no", "My sources say no", "Outlook not so good", "Very doubtful"];
```

Comment:
Nice work. By the way, .concat() also works with arrays, the 'answers' array could have been declared by concatenating the others to avoid repetition.